Lib/test/test_tools/test_msgfmt.py

```diff
@@ -1,4 +1,12 @@
-"""Tests for the Tools/i18n/msgfmt.py tool."""
+"""Tests for the Tools/i18n/msgfmt.py tool.
+
+Those tests use data files (po and mo) in the msgfmt_data folder.
```

Comment:
```suggestion
These tests use data files (po and mo) in the msgfmt_data folder.
```
Minor typo I missed, apologies!